tests/metrics/test_classification.py

```diff
@@ -155,3 +155,29 @@ def test_when_no_interactions(self) -> None:
             expected_metric_per_user,
         )
         assert np.isnan(self.metric.calc(RECO, EMPTY_INTERACTIONS, CATALOG))
+
+
+class TestHitRate:
```

Comment:
Please also add this metric to the complex test in `test_scoring.py`

Reply:
Metric landed here